features/step_definitions/additional_cli_steps.rb

```diff
@@ -4,19 +4,19 @@
 
 Then /^the output should contain all of these:$/ do |table|
   table.raw.flatten.each do |string|
-    assert_partial_output(string, all_output)
+    expect(all_commands.map { |c| c.output }.join("\n")).to include(string)
```

Comment:
How do you feel about implementing `all_output` as this? It's repeated a lot and would reduce the churn, equally if you wanted to rename it I'd be ok with that I'm more concerned about the repetition.

Reply:
Done!